components/SerialServer/src/serial.c

```diff
@@ -387,12 +387,22 @@ static void handle_char(uint8_t c)
             last_out = -1;
             statemachine = 1;
             break;
+        case 's':
+#ifdef CONFIG_DEBUG_BUILD
+            seL4_DebugDumpScheduler();
```

Comment:
I think COLOR_RESET needs to be printed first. (And this would also be why last_out is set to -1 at the end of the command - so that the next client print resets the color.